BuildResidentialHPXML/measure.rb

```diff
@@ -1418,7 +1423,7 @@ def arguments(model)
     arg.setDefaultValue(24)
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('mech_vent_total_recovery_efficiency_type', mech_vent_recovery_efficiency_type_choices, true)
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('mech_vent_recovery_efficiency_type', mech_vent_recovery_efficiency_type_choices, true)
```

Comment:
GitHub won't let me comment below, but need to add to the total/sensible recovery efficiency argument descriptions which mech vent system types they apply to.